Userland/Libraries/LibMarkdown/Text.cpp

```diff
@@ -21,9 +21,17 @@ void Text::Emph::render_to_html(StringBuilder& builder) const
     builder.append((strong) ? "</strong>" : "</em>");
 }
 
-void Text::Emph::render_for_terminal(StringBuilder&) const
+void Text::Emph::render_for_terminal(StringBuilder& builder) const
 {
-    // FIXME.
+    if (strong) {
+        builder.append("\e[1m");
+        child->render_to_html(builder);
```

Comment:
Typo? I believe this should call `render_to_terminal`.

Because of that: On which files did you test your new implementation? (I can't run and test it just as easily.)

Reply:
I tested a few man pages, and played with the rendering witht he md command. the html renderer was much more thoroughly tested with TestCommonmark. I will fix this.